niworkflows/interfaces/nibabel.py

```diff
@@ -89,3 +89,68 @@ def _run_interface(self, runtime):
         maskimg.to_filename(self._results['out_mask'])
 
         return runtime
+
+
+class _FourToThreeInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input 4d image')
+
+
+class _FourToThreeOutputSpec(TraitedSpec):
+    out_files = OutputMultiObject(File(exists=True),
+                                     desc='output list of 3d images')
+
+
+class FourToThree(SimpleInterface):
```

Comment:
I would stick with other existing interfaces conventions (i.e., ``fsl.Split``) or with the convention of ``MergeSeries`` below:
```suggestion
class SplitSeries(SimpleInterface):
```